src/Umbraco.Cms.Integrations.Automation.Zapier/Services/UserValidationService.cs

```diff
@@ -40,11 +61,11 @@ public async Task<bool> Validate(string username, string password, string userGr
 
             if (!isUserValid) return false;
 
-            if (!string.IsNullOrEmpty(userGroup))
+            if (!string.IsNullOrEmpty(_zapierSettings.UserGroup))
             {
                 var user = _userService.GetByUsername(username);
 
-                return user != null && user.Groups.Any(p => p.Name == userGroup);
+                return user != null && user.Groups.Any(p => p.Name == _zapierSettings.UserGroup);
```

Comment:
Just spotted this, but we should probably be using the user group alias rather than the name here, as that's less likely to change and then break an integration.

Reply:
Updated